### PR TITLE
Ana/fix rewards denoms in am misplaced

### LINKED ISSUE #4024: Claiming rewards denoms on ActionModal
Somehow this broke again... Going to fix it

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #4024

**Description:**

Fixes claim rewards denoms misplaced on ModalWithdrawRewards:

<h3>Before:</h3>

<h3>After:</h3>

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
e2e tests fail?

**Comment by faboweb** [comment]
@jbibla for css review

File: src/ActionModal/components/ModalWithdrawRewards.vue [hunk 1]
```diff
@@ -26,13 +26,17 @@
         }`
       "
     >
-      <div v-for="reward in totalRewards" :key="JSON.stringify(reward.denom)">
-        <span class="input-suffix">{{ reward.denom }}</span>
+      <div
+        v-for="reward in totalRewards"
+        :key="JSON.stringify(reward.denom)"
```
Comment (jbibla) [inline src/ActionModal/components/ModalWithdrawRewards.vue:31]: why do we need to stringify?

**Comment by jbibla** [comment]
ana can you share a screenshot of what the fix looks like?

**Comment by Bitcoinera** [comment]
> ana can you share a screenshot of what the fix looks like?

oh sure! My bad

File: src/ActionModal/components/ModalWithdrawRewards.vue (hunk 1 above)
Comment (Bitcoinera) [reply src/ActionModal/components/ModalWithdrawRewards.vue:31]: Good question. It was like that before

Comment (Bitcoinera) [reply src/ActionModal/components/ModalWithdrawRewards.vue]: Removed

**Pull request merged**